CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ It is based on Tidepool Blip 1.27.
 ## 3.0.6-beta - 2023-11-10
 ### Engineering use
 - YLP-2476 Use tide-whisperer v2: route /all.
+- YLP-2574 Create tool tips independently of the name of the columns
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏Given that it's more a feature than a bug, I would have added this in the `Changed` part of the changelog